src/Configuration.ts

```diff
@@ -121,10 +110,20 @@ export const WinstonLogLevel = t.keyof({
 });
 export type WinstonLogLevel = t.TypeOf<typeof WinstonLogLevel>;
 
+export const RedisConfig = t.intersection([
+  ServerConfiguration,
+  t.interface({
+    PASSWORD: t.string,
```

Comment:
the password should come from an environment variable

Reply:
I didn't got this point. It's already defined from env:

  // Redis DB Server Configuration
  REDIS_DB: {
    PORT: process.env.REDIS_DB_PORT || 6379,
    HOST: process.env.REDIS_DB_URL || "redis://localhost",
    PASSWORD: process.env.REDIS_DB_PASSWORD || "ND",
    USE_CLUSTER: process.env.REDIS_USE_CLUSTER || false
  },